components/org.wso2.identity.outbound.adapter.websubhub/src/main/java/org/wso2/identity/outbound/adapter/websubhub/util/WebSubHubAdapterConstants.java

```diff
@@ -65,7 +68,8 @@ public enum ErrorMessages {
                 "be null or empty."),
         ERROR_INVALID_WEB_SUB_OPERATION("60009", "Invalid WebSub operation input", "WebSub operation cannot be null " +
                 "or empty."),
-
+        WEB_SUB_HUB_ADAPTER_DISABLED("60010", "WebSub Hub adapter is disabled.",
+                "WebSub Hub adapter is disabled."),
```

Comment:
```suggestion
                "WebSub Hub adapter is disabled."),
```
Shall we add some context info to the error description? (Can do by embedding "%s" in the string)
Ex: `Error in topic registration. WebSub Hub adapter is disabled.`